src/main/resources/application-prod.yaml

```
@@ -0,0 +1,25 @@
+server:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)